src/cfnlint/data/DownloadsMetadata/1c9ead4af49b3a8f39632f5a30578ead5310da0b5a68ae4cf93b4be6a9a05278.meta.json

```diff
@@ -0,0 +1 @@
+{"etag": "\"f71b5ea17b03894692ac60518e30db24\"", "url": "https://s3.us-gov-west-1.amazonaws.com/cfn-resource-specifications-us-gov-west-1-prod/latest/gzip/CloudFormationResourceSpecification.json"}
```

Comment:
It makes sense to commit them, otherwise someone with the latest specs running `--update-specs` would be forced to redownload them one time to generate these files.

Reply:
> someone with the latest specs running --update-specs would be forced to redownload them one time to generate these files

Having people download the files one more time is a small price to pay IMO. I prefer to keep the codebase clean. They'll reap the benefits later :)

Reply:
I'll remove them from this PR, but I think we should consider adding them in future. If someone is using cfn-pythn-lint in a CI/CD flow where they pull the latest version each build, running `--update-specs` will force a download every time even though they might already have the latest.